app/assets/javascripts/annotations_popup.js

```diff
@@ -0,0 +1,159 @@
+const updateTweaks = () => {
```

Comment:
nit: can we call this something like `updateEditTweakButtons` (or something that makes sense with whatever `Edit` is renamed to)